src/lib/y2partitioner/dialogs/md.rb

```diff
@@ -33,6 +33,8 @@
 #   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FCustomWidget:${0}
 # @!macro [new] seeDialog
 #   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FDialog:${0}
+# @!macro [new] seeItemsSelection
+#   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FItemsSelection:${0}
```

Comment:
it looks familiar for me :) https://github.com/yast/yast-storage-ng/pull/746/files#diff-e9f6c27fd34d282cdefbf6f09306cccdR36